src/openrct2-ui/windows/News.cpp

```diff
@@ -285,11 +285,13 @@ static void window_news_scrollpaint(rct_window* w, rct_drawpixelinfo* dpi, int32
 
             switch (newsItem.Type)
             {
-                case NEWS_ITEM_RIDE:
+                case News::ItemType::Null:
+                    break;
```

Comment:
On this and other switches where you had to start handling extra enum values but not do anything, like `Null`, `Count` and `Blank`, I think we can make it clear if we group them all into a single `break`. What's your opinion @ZehMatt ?

Like
```c++
case News::ItemType::Null:
case News::ItemType::Blank:
case News::ItemType::Count:
    break;
```

Reply:
I only did it because it was failing to build on anything but Mac.  I added a default and it still failed to build on MinGW.  If there's a method of making the default work on MinGW we can just use that instead.

Reply:
I'd rather not have a `default`, because then if we ever add a new enumeration, it will compile just fine. I prefer handling the cases individually, because then compilation reminds us to look into all cases.

That being said, when there are multiple cases doing the exact same thing, in this case nothing, I think we can group them for improved readability.